locations/spiders/burger_king_ru.py

```diff
@@ -0,0 +1,32 @@
+import scrapy
+from scrapy.http import JsonRequest
+
+from locations.categories import apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+
+class BurgerKingRUSpider(scrapy.Spider):
+    name = "burger_king_ru"
+    item_attributes = {"brand": "Бургер Кинг", "brand_wikidata": "Q177054"}
+    allowed_domains = ["burgerkingrus.ru"]
+
+    def start_requests(self):
+        yield JsonRequest("https://orderapp.burgerkingrus.ru/api/v1/restaurants/search")
+
+    def parse(self, response):
+        for poi in response.json().get("response"):
+            item = DictParser.parse(poi)
+            item["phone"] = poi.get("phone", "").replace(":", "")
+            self.parse_hours(item, poi)
+            if poi.get("wifi") == "1":
+                apply_category({"internet_access": "wlan"}, item)
```

Comment:
I think this can be `apply_yes_no("internet_access=wlan", item, poi.get("wifi") == "1"`

Reply:
Nice catch, thanks! Fixed.